### PR TITLE
Initial server implementation

### PR DESCRIPTION
**Pull request by mAAdhaTTah**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,19 +3,19 @@ import logo from './logo.svg';
...
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
```
Comment (kaplanmichael) [inline src/App.js]: Will we always need to edit a page and reload first in order to get the browsersync to function?

File: src/routes/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', function (req, res) {
+    res.send(`<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
```
Comment (kaplanmichael) [inline src/routes/index.js:10]: What does the "theme-color" meta tag do?

File: src/App.js [hunk 3]
```diff
@@ -3,19 +3,19 @@ import logo from './logo.svg';
...
+    render () {
+        return (
+            <div className="App">
+                <header className="App-header">
+                    <img src={logo} className="App-logo" alt="logo"/>
+                    <h1 className="App-title">Welcome to React</h1>
+                </header>
+                <p className="App-intro">
+                    To get started, edit <code>src/App.js</code> and save to reload.
```
Comment (kaplanmichael) [inline src/App.js:14]: Will we always need to edit a page and reload first in order to get the browsersync to function?

Comment (mAAdhaTTah) [reply src/App.js:14]: It's not browserSync, and it doesn't require you to reload manually; the page will reload automatically when you save changes.

File: src/routes/index.js (hunk 2 above)
Comment (mAAdhaTTah) [reply src/routes/index.js:10]: Controls the color of the toolbar on Android. See here: https://developers.google.com/web/updates/2014/11/Support-for-theme-color-in-Chrome-39-for-Android

Came by default w/ create-react-app.

**Review by kaplanmichael — APPROVED** [review]

**Pull request merged**